DevOps/README.md

```diff
@@ -58,7 +58,6 @@
   - [Docker Tutorial - TechWorld with Nana(Youtube)](https://www.youtube.com/watch?v=3c-iBn73dDE)
   - [Learn Docker - freecodecamp(Youtube)](https://www.youtube.com/watch?v=9zUHg7xjIqQ)
   - [Docs](https://docs.docker.com/)
-  - [Complete Docker Course](https://www.youtube.com/watch?v=RqTEHSBrYFw)
   - [Complete Docker Course - From BEGINNER to PRO! (Learn Containers)](https://www.youtube.com/watch?v=RqTEHSBrYFw&t=3615s)
```

Comment:
Please add the channel name as well, as that we can avoid duplicate resources in the future.
```suggestion
  - [Complete Docker Course - From BEGINNER to PRO! (Learn Containers) - by DevOps Directive](https://www.youtube.com/watch?v=RqTEHSBrYFw&t=3615s)
```